translations/languages.json

```
@@ -6,4 +6,5 @@
   "català": "ca",
   "português brasileiro": "pt_BR",
   "русский": "ru"
```

This is not valid JSON. 
```suggestion
  "русский": "ru",
```